ui/src/lib/components/Footer.svelte

```diff
@@ -0,0 +1,72 @@
+<script>
+	let { color } = $props();
+    import greenBubbles from '$lib/images/bubbles-green.svg';
+    import pinkBubles from '$lib/images/bubbles-pink.svg';
+    import blueBubbles from '$lib/images/bubbles-blue.svg';
+
+</script>
+<style>
+    h2, h2 {
+        display: block;
+    }
+    .button.full {
+        background-color: rgba(46, 207, 158, 0.25);
+        color: #fff;
+    }
+    .button.hire {
+        padding-left: 5.8rem;
+        padding-right: 5.8rem;
+    }
+    .footer {
+        position: relative;
+        padding: 16rem 6rem 10rem;
+    }
+    .footer::before {
+        content: '';
+        display: block;
+        position: absolute;
+        bottom: 0;
+        right: 0;
+        width: 100%;
+        height: 100%;
+        background-repeat: no-repeat;
+        background-position: bottom right;
+    }
+    .footer-content {
+        display: block;
+        position: relative;
+        z-index: 1;
+    }
+    @media screen and (min-width: 600px) {
+        width: 65%;
+    }
+    .footer--blue {
+        background-image: linear-gradient(to bottom, rgba(44, 13, 93, 0), rgba(11, 54, 84, 1) 50%);
+    }
+    .footer--blue::before {
+        background-image: url('$lib/images/bubbles-blue.svg');
+    }
+    .footer--pink {
+        background-image: linear-gradient(to bottom, rgba(44, 13, 93, 0), rgba(44, 13, 93, 1) 16%);
```

Comment:
I think this gradient should be more subtle for the community page. The color change is very abrupt right now

![Screenshot from 2025-06-30 14-59-20](https://github.com/user-attachments/assets/94b4f1a4-f37d-4cc7-9206-cc1efe75828d)

It looks fine on the business page, I think it's just due to the contrast between the pink and purple.